2020-11-03 13:22:40.062 - [Info] MITM Version: 1.26
2020-11-03 13:22:40.066 - [Info] Auto Access Enabled: true
2020-11-03 13:22:40.066 - [Debug] [Init] Auto Access Barrier: true
2020-11-03 13:22:40.078 - [Info] SSH man-in-the-middle server for 172.20.0.202 listening on 0.0.0.0:10001
2020-11-03 13:22:44.509 - [Debug] [Connection] Attacker connected: 111.229.120.31 | Client Identification: SSH-2.0-libssh-0.6.3
2020-11-03 13:22:50.336 - [Debug] [Auth] Attacker 111.229.120.31 trying to authenticate with "password"
2020-11-03 13:22:50.337 - [Debug] [Auto Access] Attacker: 111.229.120.31, Threshold: 1, Attempts: 1
2020-11-03 13:22:50.337 - [Debug] [Auto Access] Compromising the honeypot
2020-11-03 13:22:50.337 - [Debug] [Auto Access] Adding the following credentials: "root:123.."
2020-11-03 13:22:50.444 - [Debug] [Auth] Valid credentials - Password Authentication
2020-11-03 13:22:50.444 - [Debug] [LXC] Attempting to connect to the honeypot: 172.20.0.202
2020-11-03 13:22:55.524 - [Debug] [Connection] Attacker connected: 101.75.215.139 | Client Identification: SSH-2.0-libssh-0.6.3
2020-11-03 13:22:57.048 - [Debug] [Auth] Attacker 101.75.215.139 trying to authenticate with "password"
2020-11-03 13:22:57.049 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2020-11-03 13:22:57.050 - [Debug] [Auth] Attacker: 101.75.215.139 has so far made 1 attempts. Remaining: 5 attempts
2020-11-03 13:22:57.572 - [Debug] [Connection] Attacker closed the connection
2020-11-03 13:22:59.713 - [Debug] [Connection] Attacker closed the connection
2020-11-03 13:23:05.916 - [Info] Exiting...
2020-11-03 13:23:05.916 - [Debug] Cleaning up...
No auth in progress
